test/function/ConsoleFunction.test.ts

```diff
@@ -15,8 +15,24 @@ describe('ConsoleFunction', () => {
         const consoleFunction = template.findResources('AWS::Lambda::Function');
         const layers = consoleFunction.Console63CA37A7.Properties.Layers;
         expect(layers).length(2);
-        expect(layers[0]).to.match(/arn:aws:lambda:us-east-1:534081306603:layer:php-81:\d+/);
+        expect(layers[1]).to.match(/arn:aws:lambda:us-east-1:534081306603:layer:php-81:\d+/);
+        expect(layers[0]).to.match(/arn:aws:lambda:us-east-1:534081306603:layer:console:\d+/);
```

Comment:
I'm looking into this but I really don't understand how this could be what we want 🤔

The test ensures that the console layer is first, but it should be last, right?

Reply:
I have been testing with `master` and it is working out fine, here is the CloudFormation template output: 

<img width="643" alt="image" src="https://github.com/brefphp/constructs/assets/720328/792a2b45-a838-4c29-b6bf-52fc01b7c5d3">

Reply:
I just tested the deployed function and everything was working correctly. TBH I don't think there is a problem to solve?

Maybe you are on a bugged CDK version?

Reply:
Update: another report here: https://github.com/brefphp/bref/discussions/1580

Not sure why I can't reproduce this, this is so weird!

Reply:
Sorry I am (and still am) on vacation, I'll try to keep a closer eye on this conversation :) 

Using the `ConsoleFunction` from the constructs yields the same result that you are seeing, deployed the layers are still reversed for me. 

Now the interesting part: If i create a lambda function myself and setting the layers  property to use php first and then console, the CF template looks the same(console first, php last), but deployed it is now in the correct order. 😕